drivers/display/display_rm67162.c

```diff
@@ -350,7 +350,7 @@ static int rm67162_write_fb(const struct device *dev, bool first_write,
 			const uint8_t *src, uint32_t len)
 {
 	const struct rm67162_config *config = dev->config;
-	uint32_t wlen = 0;
+	ssize_t wlen;
```

Comment:
I am curious why we use ssize_t in the drivers, what C spec. introduced this type?